opsdroid/events.py

```diff
@@ -115,3 +123,13 @@ def __init__(self, text, user, room, connector, raw_message=None):
         if 'thinking-delay' in self.connector.configuration:
             await self._thinking_delay()
         return await self.connector.react(self, emoji)
+
+
+class Image(Message):
```

Comment:
I think `Image` should be a subclass of `File`

Reply:
Agreed, that change will be in the next push.